packages/compass-explain-plan/package.json

```diff
@@ -35,7 +35,7 @@
     "@mongodb-js/compass-status": "^4.2.1",
     "d3": "3.5.17",
     "debug": "*",
-    "hadron-document": "^6.2.1",
+    "hadron-document": "*",
```

Comment:
Why changing this to `*`?

Reply:
that's the peer dep section, just noticed that the version here didn't match the one in the devDeps .. but wait lerna should have pick this up right? I'm going to merge it cause this should not cause a problem and try to look into these misaligned stuff later

Reply:
Lerna doesn't modify peerDeps unfortunately, no. Yeah, thinking it over it's probably even better to set it to `*` to prevent them from going out of sync, having a specific version in peers seems to be an annoyance already (see #2261 https://github.com/mongodb-js/vscode/pull/301)

Reply:
Fwiw … I *think* we’ll want to have a `"mongodb": "4.x"` peer dep in mongodb-connection-model and mongodb-data-service eventually… of course, `*` would also make this *work* :)

Reply:
Ok, thanks for the feedback you both! After this one I can put `*` for peer deps for internal deps only, for other deps I guess is fine to have any version range it makes sense as long as they align, lerna would not do anything about them anyway.

Reply:
Ah, yeah, totally makes sense!

I should've been more specific: I was thinking only about peerDeps between packages that are part of the monorepo, I think for them peerDeps are more of an annoyance than any help and maybe we can move them from peers to direct dependencies, especially taking into account that by default npm7 will be installing them